### ISSUE TITLE
Install fails on Manjaro with Python 3.9.7

### ISSUE THREAD

**Issue by mroavi** [Cimbali/pympress#233]
**Describe the bug**
Install fails on Manjaro

**To Reproduce**
```
git clone https://aur.archlinux.org/python-pympress.git
cd python-pympress
makepkg -si
```

**Expected behavior**
A successful installation.

**Environment (please complete the following information):**
 - OS: Manjaro Linux x86_64
 - Python version: 3.9.7 (default, Oct 10 2021, 15:13:22)
 - Pympress version: 998e35f git commit
 - Installation method: source

**Additional context**
```
❯ sudo makepkg -si
==> ERROR: Running makepkg as root is not allowed as it can cause permanent,
catastrophic damage to your system.

 ~/repos/python-pympress   master ?2
❯ makepkg -si
==> Making package: python-pympress 1.7.0-1 (di 07 dec 2021 02:34:21 CET)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Found v1.7.0.tar.gz
==> Validating source files with md5sums...
    v1.7.0.tar.gz ... Passed
==> Extracting sources...
  -> Extracting v1.7.0.tar.gz with bsdtar
==> Removing existing $pkgdir/ directory...
==> Starting build()...
Using setuptools.setup():
running build
running build_py
running compile_catalog
pympress/share/locale/cs/LC_MESSAGES/pympress.po
197 translated messages, 58 untranslated messages.
pympress/share/locale/es/LC_MESSAGES/pympress.po
195 translated messages, 60 untranslated messages.
pympress/share/locale/fr/LC_MESSAGES/pympress.po
251 translated messages, 4 untranslated messages.
pympress/share/locale/pl/LC_MESSAGES/pympress.po
146 translated messages, 5 fuzzy translations, 104 untranslated messages.
pympress/share/locale/de/LC_MESSAGES/pympress.po
166 translated messages, 89 untranslated messages.
==> Entering fakeroot environment...
==> Starting package()...
Using setuptools.setup():
running install
running build
running build_py
running compile_catalog
pympress/share/locale/cs/LC_MESSAGES/pympress.po
pympress/share/locale/cs/LC_MESSAGES/pympress.po:1:2: syntax error
pympress/share/locale/cs/LC_MESSAGES/pympress.po:1: keyword "L" unknown
pympress/share/locale/cs/LC_MESSAGES/pympress.po:2: keyword "b" unknown
pympress/share/locale/cs/LC_MESSAGES/pympress.po:3: keyword "V" unknown
pympress/share/locale/cs/LC_MESSAGES/pympress.po:8: keyword "E" unknown
pympress/share/locale/cs/LC_MESSAGES/pympress.po:9: end-of-line within string
pympress/share/locale/cs/LC_MESSAGES/pympress.po:10: end-of-line within string
pympress/share/locale/cs/LC_MESSAGES/pympress.po:11: end-of-line within string
pympress/share/locale/cs/LC_MESSAGES/pympress.po:13: keyword "h" unknown
pympress/share/locale/cs/LC_MESSAGES/pympress.po:16: keyword "S7" unknown
pympress/share/locale/cs/LC_MESSAGES/pympress.po:17: keyword "y7" unknown
pympress/share/locale/cs/LC_MESSAGES/pympress.po:18: end-of-line within string
pympress/share/locale/cs/LC_MESSAGES/pympress.po:19: keyword "P" unknown
pympress/share/locale/cs/LC_MESSAGES/pympress.po:21: keyword "J" unknown
pympress/share/locale/cs/LC_MESSAGES/pympress.po:22: keyword "d" unknown
pympress/share/locale/cs/LC_MESSAGES/pympress.po:23: keyword "slide" unknown
pympress/share/locale/cs/LC_MESSAGES/pympress.po:24: keyword "Content" unknown
pympress/share/locale/cs/LC_MESSAGES/pympress.po:25: keyword "Content" unknown
pympress/share/locale/cs/LC_MESSAGES/pympress.po:26: keyword "X" unknown
pympress/share/locale/cs/LC_MESSAGES/pympress.po:27: keyword "Project" unknown
msgfmt: too many errors, aborting
Traceback (most recent call last):
  File "/home/mroavi/repos/python-pympress/src/pympress-1.7.0/setup.py", line 271, in <module>
    setuptools.setup(**options)
  File "/usr/lib/python3.9/site-packages/setuptools/__init__.py", line 153, in setup
    return distutils.core.setup(**attrs)
  File "/usr/lib/python3.9/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "/usr/lib/python3.9/distutils/dist.py", line 966, in run_commands
    self.run_command(cmd)
  File "/usr/lib/python3.9/distutils/dist.py", line 985, in run_command
    cmd_obj.run()
  File "/usr/lib/python3.9/site-packages/setuptools/command/install.py", line 61, in run
    return orig.install.run(self)
  File "/usr/lib/python3.9/distutils/command/install.py", line 546, in run
    self.run_command('build')
  File "/usr/lib/python3.9/distutils/cmd.py", line 313, in run_command
    self.distribution.run_command(command)
  File "/usr/lib/python3.9/distutils/dist.py", line 985, in run_command
    cmd_obj.run()
  File "/usr/lib/python3.9/distutils/command/build.py", line 135, in run
    self.run_command(cmd_name)
  File "/usr/lib/python3.9/distutils/cmd.py", line 313, in run_command
    self.distribution.run_command(command)
  File "/usr/lib/python3.9/distutils/dist.py", line 985, in run_command
    cmd_obj.run()
  File "/home/mroavi/repos/python-pympress/src/pympress-1.7.0/setup.py", line 96, in run
    self.distribution.run_command('compile_catalog')
  File "/usr/lib/python3.9/distutils/dist.py", line 985, in run_command
    cmd_obj.run()
  File "/home/mroavi/repos/python-pympress/src/pympress-1.7.0/setup.py", line 84, in run
    subprocess.check_output(cmd)
  File "/usr/lib/python3.9/subprocess.py", line 424, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/usr/lib/python3.9/subprocess.py", line 528, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['msgfmt', '--statistics', '--use-fuzzy', 'pympress/share/locale/cs/LC_MESSAGES/pympress.po', '-o', 'pympress/share/locale/cs/LC_MESSAGES/pympress.po']' returned non-zero exit status 1.
==> ERROR: A failure occurred in package().
    Aborting...

```

**Comment by Cimbali** [comment]
Yes, this is rather unfortunate. The AUR package is not maintained by me, and it really should be getting the sources from pypi, rather than github. That way the compiling of translation catalogs is already done, amongst other things. Other workarounds include editing the package to use the current master release (the building is fixed there), or the easiest, to install python-babel.

Also be sure to install dependencies that are not listed in this package, i.e. `poppler-glib`, and for media support, the gstreamer packages.

**Comment by mroavi** [comment]
Thanks for the quick answer @Cimbali. Could you give me more info on how to use the workarounds that you mention above?
I suppose that the first workaround, which is making the AUR package get the source from pypi rather than github, is out of my reach. Right? If not, could you explain to me how to do this?
The second workaround is to use the current master release. How can I do this? Do I have to modify the PKGBUILD file?
The third one is to install python-babel. I installed python-babel using pip (`pip install Babel`). I also installed poppler-glib (`sudo pacman -S poppler-glib`). Then I repeated the steps described above and got the same error.

**Comment by Cimbali** [comment]
Strange that installing Babel did not work, that’s what [comments on the AUR package](https://aur.archlinux.org/packages/python-pympress/#comments) report.

To modify the install you could do `makepkg --nobuild`, patch the sources, then `makepkg --noextract` to use the patched sources.

I think the change is simply to replace `mo = po.with_suffix('.po')` on line 76 of setup.py with `mo = po.with_suffix('.mo')` from #232. Or just replace the sources by extracting [the pypi sources (zip)](https://files.pythonhosted.org/packages/c0/65/041a4feb4d432edce8215703892eef5379d0d925c7f304332501c29ddfac/pympress-1.7.0.tar.gz#sha256=0311f43f2016604108a90031f601b6798c973228cb64666a5e446195ddf689e1) or [current master sources (zip)](https://github.com/Cimbali/pympress/archive/refs/heads/master.zip) in that directory.

**Comment by Cimbali** [comment]
I’ve released the build fix in v1.7.1.
